src/aggregate/aggregate_request.c

```diff
@@ -904,9 +893,7 @@ static ResultProcessor *getScorerRP(AREQ *req) {
   }
   ScoringFunctionArgs scargs = {0};
   ExtScoringFunctionCtx *fns = Extensions_GetScoringFunction(&scargs, scorer);
-  if (!fns) {
-    fns = Extensions_GetScoringFunction(&scargs, DEFAULT_SCORER_NAME);
-  }
+  assert(fns);
```

Comment:
would this crash on invalid input then?

Reply:
Unknown input is checked before, if we reach here input must be valid

Reply:
Why remove the assert? assert its a good think to have in the code, better to catch it here then after ..